storages/userStorage.js

```diff
@@ -39,7 +39,15 @@ class UserStorage {
     try {
       const query = "SELECT * FROM users WHERE email = ?";
       const [results] = await db.query(query, [userEmail]);
-      return results[0] || null;
+      const camelCaseResults = results.map((row) => {
+        return {
+          id: row.id,
+          email: row.email,
+          profileImg: row.profile_image,
+          name: row.name,
+        };
+      });
```

Comment:
```suggestion
    const row = results[0];
    const camelCaseResult = {
      id: row.id,
      email: row.email,
      profileImg: row.profile_image,
      name: row.name,
    };
```
getUserInfo 메서드에서 map을 사용하여 배열을 처리하는 부분에서,,,,,
이메일로 검색된 사용자가 한 명이라면 직접 results[0]에서 첫 번째 사용자 정보를 추출하여 반환하는 게 효율적일수도 있을 것 같습니다.

Reply:
기존에 `results`라는 변수로 컨벤션을 정하신 건지는 잘 모르겠습니다만
변수명을 `camelCaseResults`라고 짓는 것보다 **"이 데이터가 무엇을 의미하는지"** 를
명확하게 나타내는 것이 더 중요하다고 생각합니다.😊

추천을 좀 드려보자면...
userInfo, users, formattedUsers
이렇게 될 것 같습니당

Reply:
확인했습니다 !!